ts/src/trycp/conductor/scenario.ts

```diff
@@ -156,22 +156,32 @@ export class TryCpScenario {
                 agentsApps: appOptions,
               });
               const adminWs = conductor.adminWs();
-              const { port } = await adminWs.attachAppInterface();
-              await conductor.connectAppInterface(port);
               const players: TryCpPlayer[] = await Promise.all(
-                appInfos.map((appInfo) =>
-                  conductor
-                    .connectAppAgentWs(port, appInfo.installed_app_id)
-                    .then((appAgentWs) =>
-                      enableAndGetAgentApp(adminWs, appAgentWs, appInfo).then(
-                        (agentApp) => ({
-                          conductor,
-                          appAgentWs,
-                          ...agentApp,
-                        })
+                appInfos.map(async (appInfo) => {
+                  const { port } = await adminWs.attachAppInterface();
+                  const issued = await adminWs.issueAppAuthenticationToken({
+                    installed_app_id: appInfo.installed_app_id,
+                  });
+                  // This doesn't make a lot of sense... but we are asking the trycp server to create a connection,
```

Comment:
Yeah, I guess a TryCP socket relay conundrum, like the attach and connect interface.

Reply:
Exactly that, it's not faithfully implementing the interface but actually doing something subtly different that is now very obvious from outside :/